test/test_gpu_utils.cpp

```diff
@@ -59,6 +59,11 @@ TEST_F(NVFuserTest, FusionMergeDims_CUDA) {
   }
   std::vector<size_t> expect_dims{0, 1, 2, 2, 3, 4, 5, 2, 2, 2, 6};
   EXPECT_EQ(dims, expect_dims);
+
+  auto merged_dim = tv->axis(2)->toString(0);
+  EXPECT_EQ(
+      merged_dim.substr(merged_dim.find("{")),
+      "{( 23 * ( 19 * ( 29 * ( 5 * 7 ) ) ) )}");
```

Comment:
lol is there a better way to check the order of merged IterDomain than string comparison? @zasdfgbnm

Reply:
Maybe you could use `merged_dim->definition()->outer()` and `merged_dim->definition()->inner()`. Or you could use `IterVisitor::getInputsTo()`.